### PR TITLE
New doc draft: HTTPS for sites using customized base domains

### LINKED ISSUE #898: New Doc Proposal - HTTPS for Sites Using Customized Base Domains
This new doc has been requested internally. @joshkoenig can you review the outline?
ping @bmackinney @nataliejeremy 

## Title
HTTPS for Sites Using Customized Base Domains

## Description
Learn how to enable HTTPS for site environments using your Organization's customized base domain. 

### Intro
Feature is available to Pantheon Partners, Strategic Partners, Enterprise accounts, Resellers, and OEM Partners service levels. 
Restate customized base domain purpose and value. Link to [Customizing Your Base Domain](https://pantheon.io/docs/articles/organizations/base-domains/#request-the-base-domain).
### Before You Begin
- Must have a customized base domain
 - If you do not have a customized base domain, [request one](https://pantheon.io/docs/articles/organizations/base-domains/#request-the-base-domain) before proceeding
- Requires Wildcard cert from a recommended SSL provider

### Single Domain Configuration
In single domain config scenarios, sites associated with the organization (e.g. projectsite-env.pantheonpartner.com) and the main site (e.g. www.pantheonpartner.com) will be served using the same wildcard certificate. 
Requires DNS change for pantheonpartner.com - set up all DNS to go to the IP for the TLS/SSL endpoint in the live env, like so:
```
*.pantheonpartner.com A [IP for TLS/SSL endoint in live env]
```
Sites associated with the org will be accessible via a similar URL to the following: `https://www.projectsite-env.pantheonpartner.com`
### Multiple Domain Configuration
Alternatively, you could load up one cert for the live environment, and use the test environment for another cert (e.g. "*.partnerprojects.com") that is on a more clear "dev" domain, configuring the different DNS records accordingly. 
## Expected Audience
Partners 
## Path
 `/docs/articles/organizations/https-for-sites-using-customized-base-domains`

### PR DESCRIPTION
**Pull request by rachelwhitton**
Closes #898 

 Draft **Enable HTTPS for Sites Using Customized Base Domains**

@joshkoenig @bmackinney can you review?

### CODE REVIEW

**Comment by rachelwhitton** [comment]
ping @joshkoenig

**Comment by joshkoenig** [comment]
My bad! I'll review this evening.

File: source/docs/articles/organizations/https-for-sites-using-customized-base-domains.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+## Before You Begin
+Be sure that you have:
+
+- A customized base domain, otherwise  [request one](/docs/articles/organizations/base-domains/#request-the-base-domain) and [consider the effects](/docs/articles/organizations/base-domains/#effects-and-considerations) before proceeding
+- A Wildcard SSL Certificate from an [SSL Provider](/docs/articles/sites/domains/adding-a-ssl-certificate-for-secure-https-communication#ssl-providers)
+
+## Create an HTTPS endpoint
+[Add a Wildcard SSL Certificate](/docs/articles/sites/domains/adding-a-ssl-certificate-for-secure-https-communication#steps-to-enable-ssl) to your main site's Live environment. This will create an HTTPS endpoint for all sites utilizing your base domain. Sites associated with the organization (e.g. `projectsite-env.pantheonpartner.com`) and the main site (e.g. `www.pantheonpartner.com`) will be served using the same Wildcard certificate.
```
Comment (joshkoenig) [inline source/docs/articles/organizations/https-for-sites-using-customized-base-domains.md]: I feel like this starts off wrong. What is my main site? Why am I adding a wildcard there? This section should be re-done and merged with the "development domains" section below. I would approach it based on whether or not the customer wants to use their primary domain name ("*.mydomain.com") or have a dedicated domain name ("*.mypantheonprojects.com") for their projects. 

That can be followed by the note that Partners shouldn't have to pay for this since they get a site for free.

Finally, we can end with DNS instructions. They should say something like "Direct the wildcard DNS to the IP address associated with the wildcard cert. This can be found within the **Domains/SSL** tab for the environment where your wildcard certificated was loaded."

Comment (rachelwhitton) [inline source/docs/articles/organizations/https-for-sites-using-customized-base-domains.md]: Awesome, thanks for the feedback @joshkoenig! Just updated, can you take another look? 

Also ping @bmackinney and @nataliejeremy for review.

File: source/docs/articles/organizations/https-for-sites-using-customized-base-domains.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- A Wildcard SSL Certificate from an [SSL Provider](/docs/articles/sites/domains/adding-a-ssl-certificate-for-secure-https-communication#ssl-providers)
+
+## Create HTTPS Endpoint
+Configuring your customized base domain as an HTTPS endpoint enables all environment URLs (including Multidev) associated with your organization to be accessible via HTTPS. This requires adding a wildcard certificate to the base domain in use, which is either a subdomain of your primary site (`sites.mydomain.com`) or a dedicated domain name (`mypantheonprojects.com`).
+<div class="alert alert-info">
+<h4>Note</h4>
+<strong>Pantheon Partners:</strong> Running your agency's primary site on Pantheon is <strong>free</strong>, as is the option to add an SSL. This means you can use your primary site as an HTTPS endpoint at no cost. This offer is available to Pantheon Partner organizations exclusively.
+</div>
+Organizations using a subdomain as their customized base domain (`sites.mydomain.com`) must enable a wildcard certificate within the primary site's Live environment to create an HTTPS endpoint for associated sites.
```
Comment (joshkoenig) [inline source/docs/articles/organizations/https-for-sites-using-customized-base-domains.md]: I think it's important that we're clear about where the wildcard is at, and we can be a little more precise with the wording. How about:

"Organizations using a subdomain as their customized base domain (`*.mydomain.com`) may enable a wildcard certificate within the primary site's Live environment. This allows the same HTTPS endpoiny to serve for both their Prumary site's live environment (e.g. `mydomain.com` and `www.mydomain.com`), and for their custom subdomains: `site1.mydomain.com`, `site2.mydomain.com`, and so on."

File: source/docs/articles/organizations/https-for-sites-using-customized-base-domains.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Organizations using a dedicated domain name (`mypantheonprojects.com`) must enable two individual certificates, one within the primary site's Live environment and one wildcard certificate within the site's Test environment. The Test environment will serve as the HTTPS endpoint for associated projects.
```
Comment (joshkoenig) [inline source/docs/articles/organizations/https-for-sites-using-customized-base-domains.md]: I'd reword the intro slightly: "Organizations using a dedicated domain name for their Pantheon custom subdomain (`*.mypantheonprojects.com`) will need to use two different certificates..."

**Comment by joshkoenig** [comment]
Looking pretty good. I added a couple minor comments, but with those changes I think we are good to go. :+1:

**Comment by rachelwhitton** [comment]
Thanks for the help @joshkoenig!! Much appreciated
